keyboards/acheron/doddle60/keymaps/via/config.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
+
+#define DYNAMIC_KEYMAP_LAYER_COUNT 2
```

Comment:
At the moment, 6 layers would actually fit here. But running the defaults would be my suggestion, which would be to delete this file.

Reply:
File deleted in 44d8e74da